spec/spec_helper.rb

```diff
@@ -14,11 +14,11 @@
 require 'rom-factory'
 require 'rspec'
 
-Dir[root.join('support/*.rb').to_s].each do |f|
+Dir[root.join('support/*.rb').to_s].sort.each do |f|
   require f
 end
 
-Dir[root.join('shared/*.rb').to_s].each do |f|
+Dir[root.join('shared/*.rb').to_s].sort.each do |f|
```

Comment:
Unfortunately still necessary until support for 2.7 is removed. Deterministic globbing [was added to 3.0](https://bugs.ruby-lang.org/issues/8709).

Reply:
that's ok to keep it